packages/frontend/src/features/semanticViewer/components/ContentNoResults.tsx

```diff
@@ -0,0 +1,92 @@
+import { Box, Tabs } from '@mantine/core';
+import { IconCodeCircle } from '@tabler/icons-react';
+import { useState, type FC } from 'react';
+import { Panel, PanelGroup, PanelResizeHandle } from 'react-resizable-panels';
+import MantineIcon from '../../../components/common/MantineIcon';
+import SuboptimalState from '../../../components/common/SuboptimalState/SuboptimalState';
+import SqlViewer from './SqlViewer';
+
+enum TabPanel {
+    SQL = 'SQL',
+}
+
+const ContentNoResults: FC = () => {
```

Comment:
do you think we need to create new component for this or is it possible to fit it in `ContentResults`